package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "jaas-dashboard",
+  "description": "A dashboard for JAAS (Juju as a service)",
+  "version": "0.0.1",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/canonical-web-and-design/jaas-dashboard.git"
+  },
+  "keywords": [
+    "juju",
+    "jaas",
+    "canonical",
+    "ubuntu"
+  ],
+  "author": "Jeff Pihach (https://fromanegg.com)",
+  "license": "LGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/canonical-web-and-design/jaas-dashboard/issues"
+  },
+  "homepage": "https://github.com/canonical-web-and-design/jaas-dashboard#readme",
+  "scripts": {
+    "build": "react-scripts build",
+    "clean": "./scripts/clean-files",
+    "serve": "yarn run start",
+    "start": "react-scripts start",
+    "test": "react-scripts test",
+    "watch": ""
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "dependencies": {
+    "node-sass": "^4.12.0",
```

Comment:
You should probably just pin these dependencies now, otherwise renovate will be angry with you.

Reply:
Done